src/services/TestDataService.ts

```diff
@@ -1234,6 +1235,54 @@ export class TestDataService {
         return salesChannel;
     }
 
+    /**
+     * Creates a new domain for a sales channel.
+     * 
+     * @param overrides - Specific data overrides that will be applied to the sales channel domain data struct.
+     */
+    async createSalesChannelDomain(overrides: Partial<SalesChannelDomain> = {}): Promise<SalesChannelDomain>  {
+        const salesChannelId = this.defaultSalesChannel.id;
+        const currencyId = this.defaultCurrencyId;
+        const languageId = this.defaultLanguageId;
+        const snippetSetId = await getSnippetSetId('en-GB', this.AdminApiClient);
+
+        const salesChannelDomainStruct = this.getSalesChannelDomainStruct(salesChannelId, currencyId, languageId, snippetSetId, overrides);
+
+        const response = await this.AdminApiClient.post(`sales-channel-domain?_response=detail`, {
+            data: salesChannelDomainStruct,
+        });
+
+        expect(response.ok()).toBeTruthy();
+
+        const { data: salesChannelDomain } = (await response.json()) as { data: SalesChannelDomain };
+
+        this.addCreatedRecord('sales_channel_domain', salesChannelDomain.id);
+
+        return salesChannelDomain;
+    }   
+
+    getSalesChannelDomainStruct(
```

Comment:
could you please put that method to the area where all other get methods are located in the TestDataService?